### PR TITLE
First pass at setting up testing frameworks karma, mocha, chai

### PR DESCRIPTION
**Pull request by prushforth**
@Malvoz what do you think?  Do you have experience with these tools?

### CODE REVIEW

**Comment by prushforth** [comment]
@ahmadayubi have a look at the source of this PR, maybe pull it into your own repo.  I am working on setting up testing, using karma, mocha, chai and a whole host of other herbs and spices.  Maybe playright will be integratable at some point.  Next step for me is getting this to do checks in travis, perhaps with GitHub Actions tbd.

**Comment by Malvoz** [comment]
@prushforth I do not, no, sorry.

**Comment by prushforth** [comment]
@malvoz me either😊! Anyway I just wanted to keep up with our process of PR approval. I'll try not to bug you too much 😉

**Comment by Malvoz** [comment]
Don't worry, ping me any time. If I can help - that's great, if not  - perhaps others will 🙏

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 # Customized built-in &lt;map&gt; element
 
-[![Build Status](https://travis-ci.org/prushforth/Web-Map-Custom-Element.svg?branch=master)](https://travis-ci.org/prushforth/Web-Map-Custom-Element)
+[![Build Status](https://travis-ci.org/Maps4HTML/Web-Map-Custom-Element.svg?branch=master)](https://travis-ci.org/prushforth/Web-Map-Custom-Element)
 
-The customized built-in `<map>` element is a prototype [implementation](http://maps4html.github.io/Web-Map-Custom-Element/)
+The customized built-in `<map>` element is a prototype [implementation](https://travis-ci.org/github/Maps4HTML/Web-Map-Custom-Element)
```
Comment (Malvoz) [inline README.md:6]: @prushforth was the change of link destination here intended?

Comment (prushforth) [reply README.md:6]: No it was a mistake. Thanks for picking that up. Will fix.

**Pull request merged**